sidekiq/README.md

```diff
@@ -57,8 +57,24 @@ No additional installation is needed on your server.
        mappings:
          - match: "sidekiq.sidekiq.*"
            name: "sidekiq.*"
```

Comment:
```
         - match: "sidekiq.sidekiq.*"
           name: "sidekiq.*"
```

What does this rule intend to match?

Reply:
This is for naming purposes, so that metrics aren't named something like: `sidekiq.sidekiq.successes `

Reply:
Ok, then you probably need that:

```
         - match: "sidekiq.sidekiq.*"
           name: "sidekiq.$1"
```